src/test/java/org/openmrs/performance/http/DoctorHttpService.java

```diff
@@ -149,22 +149,33 @@ public HttpRequestActionBuilder getAllActiveOrders(String patientUuid) {
 		    "/openmrs/ws/rest/v1/order?patient=" + patientUuid + "&careSetting=" + CARE_SETTING_UUID + "&status=ACTIVE");
 	}
 
-	public HttpRequestActionBuilder getDrugOrders(String patientUuid) {
+
+	public HttpRequestActionBuilder getDrugOrdersExceptCancelledAndExpired(String patientUuid) {
 		String customRepresentation = """
-		        custom:(uuid,dosingType,orderNumber,accessionNumber,
-		        	patient:ref,action,careSetting:ref,previousOrder:ref,dateActivated,scheduledDate,dateStopped,autoExpireDate,
-		        	orderType:ref,encounter:ref,
-		        	orderer:(uuid,display,person:(display)),
-		        	orderReason,orderReasonNonCoded,orderType,urgency,instructions,
-		        	commentToFulfiller,
-		        	drug:(uuid,display,strength,
-		        		dosageForm:(display,uuid),concept),
-		        		dose,doseUnits:ref,
-		        	frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,numRefills,dosingInstructions,
-		        	duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)
-		        """;
-		return http("Get Orders").get("/openmrs/ws/rest/v1/order" + "?patient=" + patientUuid + "&careSetting="
-		        + CARE_SETTING_UUID + "&status=any&orderType=" + DRUG_ORDER + "&v=" + customRepresentation);
+        custom:(uuid,dosingType,orderNumber,accessionNumber,patient:ref,action,careSetting:ref,
+        previousOrder:ref,dateActivated,scheduledDate,dateStopped,autoExpireDate,orderType:ref,
+        encounter:ref,orderer:(uuid,display,person:(display)),orderReason,orderReasonNonCoded,
+        orderType,urgency,instructions,commentToFulfiller,drug:(uuid,display,strength,dosageForm:(display,uuid),concept),
+        dose,doseUnits:ref,frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,
+        numRefills,dosingInstructions,duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)
+        """;
+		return http("Get Drug Orders except the cancelled and expired").get(
+				"/openmrs/ws/rest/v1/order" + "?patient=" + patientUuid + "&careSetting=" + CARE_SETTING_UUID
+						+ "&status=any&orderType=" + DRUG_ORDER + "&excludeCanceledAndExpired=true&v=" + customRepresentation);
+	}
+
+	public HttpRequestActionBuilder getDrugOrdersExceptDiscontinuedOrders(String patientUuid) {
+		String customRepresentation = """
+        &v=custom:(uuid,dosingType,orderNumber,accessionNumber,patient:ref,action,careSetting:ref,
+        previousOrder:ref,dateActivated,scheduledDate,dateStopped,autoExpireDate,orderType:ref,
+        encounter:ref,orderer:(uuid,display,person:(display)),orderReason,orderReasonNonCoded,
+        orderType,urgency,instructions,commentToFulfiller,drug:(uuid,display,strength,dosageForm:(display,uuid),concept),
+        dose,doseUnits:ref,frequency:ref,asNeeded,asNeededCondition,quantity,quantityUnits:ref,
+        numRefills,dosingInstructions,duration,durationUnits:ref,route:ref,brandName,dispenseAsWritten)
+        """;
```

Comment:
@Bawanthathilan this looks like a formatting issue. I'm not sure why the validator didn't catch this. Any idea?